e2e/cypress/integration/emoji/sorted_emojis_spec.js

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/**
+ * Get last recently used emoji and makes sure has given ID
+ * @param {emojiId} emojiId - emoji id you want to make sure was the las recently used
+ */
+function assertLastUsedEmojiHasId(emojiId) {
+    cy.get('.emoji-picker__item').eq(0).children('div').children('img').should('have.id', emojiId);
+}
+
+/**
+ * Compare 2 arrays and return true if the are identical
+ * @param {arr1, arr2}
+ */
+function arraysAreIdentical(arr1, arr2) {
```

Comment:
See comment below about this custom function.

Reply:
done